### PR TITLE
Open FakeDatabaseManager.kt

### PR DESCRIPTION
**Pull request by yuangu-google**

### CODE REVIEW

File: java/arcs/jvm/storage/database/testutil/FakeDatabaseManager.kt [hunk 1]
```diff
@@ -39,7 +39,7 @@ import kotlinx.coroutines.sync.Mutex
 import kotlinx.coroutines.sync.withLock
 
 /** [DatabaseManager] which generates fake [Database] objects. */
-class FakeDatabaseManager : DatabaseManager {
+open class FakeDatabaseManager : DatabaseManager {
```
Comment (galganif) [inline java/arcs/jvm/storage/database/testutil/FakeDatabaseManager.kt:42]: you also need to open the function you want to override: getEntitiesCount getStorageSize isStorageTooLarge
by the way, I think you can just do this changes in your cl, does not need to be a separate pr

**Comment by bgogul** [comment]
The changes in this PR were committed in https://github.com/PolymerLabs/arcs/commit/cee3c2f963b1e5d27c95cd81a62e0582a3fa57ee

**Comment by bgogul** [comment]
Closing this PR as the changes were committed in https://github.com/PolymerLabs/arcs/commit/cee3c2f963b1e5d27c95cd81a62e0582a3fa57ee
